redaxo/src/addons/be_style/plugins/customizer/package.yml

```diff
@@ -0,0 +1,14 @@
+package: be_style/customizer
+version: '5.0-alpha7'
+author: 'Peter Thiel'
+supportpage: www.redaxo.org/de/forum/
+compile: 0
```

Comment:
Damit kann das kompilieren der sass Dateien gestartet werden.
Wird aber nur in `be_style` und `be_style/redaxo` verwendet.